jobs/rocoto/aeroanlfinal.sh

```diff
@@ -1,12 +1,15 @@
 #! /usr/bin/env bash
 
-source "$HOMEgfs/ush/preamble.sh"
+source "${HOMEgfs}/ush/preamble.sh"
 
 ###############################################################
 # Source FV3GFS workflow modules
-. $HOMEgfs/ush/load_fv3gfs_modules.sh
+. "${HOMEgfs}/ush/load_fv3gfs_modules.sh"
```

Comment:
g-w PR #1091 replaced `load_fv3gfs_modules.sh` with `load_ufsda_modules.sh`.   With this change we do not need to load modules in the job executed by this rocoto driver.   We can consider making a similar change to the other rocoto jobs in this PR.